packages/api/src/server/http/routes.ts

```diff
@@ -3,10 +3,10 @@ import {cache} from './../../cache.js'
 
 const router = new Router()
 
-router.get('/top-100', async (ctx) => (ctx.body = cache.bluelist))
+router.get('/top100', async (ctx) => (ctx.body = cache.bluelist))
```

Comment:
To avoid work in the future yes maybe but on the other end we not gonna have it open for devs to use it so API will always match front I guess.

Reply:
yes it's good for now

Reply:
i was thinking if one day we move the PWA to your server `blue.leofcoin.org` maybe we want to keep paths clean, because in general PWA redirect all pathes to the main index.html except the api routes

Reply:
Yes but it's pwa so not really needed, also not sure if it would be in the leofcoin domain so we can do `domain.com` for front and `api.domain.com` for api?

Reply:
yeah right